endoreg_db/urls.py

```
@@ -1,75 +1,83 @@
     CenterViewSet,
     start_examination,
     get_location_choices,
     get_morphology_choices,     
 )
-from .views.Frames_NICE_and_PARIS_classifications_views import (
+from .views.annotation.Frames_NICE_and_PARIS_classifications_views import (
     ForNiceClassificationView, 
     ForParisClassificationView,
     BatchClassificationView,
     ClassificationStatusView
 )
```

Pylint (3.3.7)

[error] No name 'annotation' in module 'endoreg_db.views'

(E0611)